src/archivematicaCommon/tests/test_elasticsearch_functions.py

```diff
@@ -88,3 +89,116 @@ def test_set_tags(self):
     def test_set_tags_fails_when_file_cant_be_found(self):
         with pytest.raises(elasticSearchFunctions.EmptySearchResultError):
             elasticSearchFunctions.set_file_tags(self.client, 'no_such_file', [])
+
+    @mock.patch('elasticSearchFunctions.get_dashboard_uuid')
+    @mock.patch('elasticSearchFunctions.wait_for_cluster_yellow_status')
+    @mock.patch('elasticSearchFunctions.try_to_index')
+    def test_index_mets_file_metadata(
+            self,
+            dummy_try_to_index,
+            dummy_wait_for_cluster_yellow_status,
+            dummy_get_dashboard_uuid,
```

Comment:
Yes, this is something I always have trouble being consistent with. I prefer the single line approach because it makes a bit easier to spot changes in diffs. When you have multiple identifiers in a single line and you change or delete only one identifier you need to scan the whole line. I also add commas after the last parameters in case you add extra parameters later, again making the diff easier to read. Hopefully using `black` will put us all in sync :)